api-client-util/src/main/java/com/takipi/api/client/util/regression/RegressionInput.java

```diff
@@ -23,8 +23,10 @@
 	public String viewId;
 
 	public DateTime activeWindowStart;
+	public DateTime nonRegressionActiveWindowStart;
```

Comment:
I don't see where this is used.
Is it used someplace outside of the `api-client`?

Reply:
Lingering garbage I forgot to remove.